middle.mkd

```diff
@@ -274,8 +274,9 @@ A GeoJSON object with the type "Feature" is a feature object.
 A GeoJSON object with the type "FeatureCollection" is a feature
 collection object. A feature collection object MUST have a member with
 the name "features". The value of "features" is an array. Each element
-of the array is a feature object as defined above. It is possible for
-this array to be empty.
+of the array MUST be a feature object as defined above. It is possible
+for this array to be empty. The values in the array are ordered, meaning
+that GeoJSON implementations MUST maintain order when processing them.
```

Comment:
> The values in the array are ordered

Since a JSON array is "an *ordered sequence* of zero or more values", I don't think it adds value to our spec to say that the values in the array are ordered.

> GeoJSON implementations MUST maintain order when processing them

I'm curious what is motivating this change.